doc/mgr/dashboard.rst

```diff
@@ -74,6 +89,9 @@ will be stored in the MON's configuration database::
 The password will be stored in the configuration database in encrypted form
```

Comment:
Clearly, the command two lines before this comment is wrong. But I'm not sure what the correct version is. Maybe this?

```
ceph tell mgr.\* set-login-credentials <username> <password>
```

?

Reply:
It's `ceph mgr dashboard set-login-credentials <username> <password>` - added a commit to fix.

Reply:
@smithfarm somehow I don't seem to see this last commit in the current PR. Did you push it?

Reply:
@LenzGr I removed the commit because the command currently in the file `ceph dashboard set-login-credentials <username> <password>` is correct. The problem was I couldn't get the command to work for me, due to a bug since fixed by @rjfd.